src/components/MapContainer/Map/Map.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useRef, useState, useEffect } from 'react'
+import './Map.css'
+import MapContext from './MapContext'
+import * as ol from 'ol'
+
+interface props {
+  children?: React.ReactNode
+  zoom: number
+  center: number[]
+}
+
+const Map: React.FC<props> = ({ children, zoom, center }: props) => {
+  const mapRef = useRef() as React.MutableRefObject<HTMLInputElement>
+  const [map, setMap] = useState({})
+
+  // on component mount, create map object, set map state, pass map state into mapcontext for access in layers.
+  useEffect(() => {
+    const options = {
+      view: new ol.View({ zoom, center }),
+      layers: [],
+      controls: [],
+      overlays: [],
+    }
+
+    const mapObject = new ol.Map(options)
+    mapObject.setTarget(mapRef.current)
+    setMap(mapObject)
+
+    return () => mapObject.setTarget(undefined)
+  }, [zoom, center])
+
+  // zoom change handler
```

Comment:
do we need these comments?

Reply:
I removed them for now, I have the dummy application if I need to put them back in. I figured for now we are just setting a singular map centered around the sound and doing several vector layers.